client/battle/CBattleInterface.h

```diff
@@ -1,407 +1,404 @@
-/*

```

Comment:
Whole file changed, line ending problem?

Reply:
I made small test, by using push -f I checked for LF, CRLF, CR, but it's not resolve the problem, any ideas what it can be?